### ISSUE TITLE
New firmware feedback

### ISSUE THREAD

**Issue by s3kn0tr0n1c** [rasteri/SC1000#8]
Thanks Rasteri for addressing the issues with the new firmware. This thing totally rocks now. Loving the custom settings you can choose from. Ive gave mine a good cut session with new firmware and all my issues have been resolved. Massive thanks.

I did notice a couple of times when fader was open(sample on) it sounded a little robotic, sort of like the sound I got on original firware when fader was in the cut zone. Moving the fader back to "off" cleared it. This only happened a couple of times in quite a long session so Im not too bothered tbh. All in all really good work. This defo the best portable scratching device out ther now.

Ive done some cut session videos. Feel free to share them on any of your SC1000 pages on social media. I dont do social media or I'd post them in there.

https://www.youtube.com/watch?v=ECYCsCCTnXI

https://www.youtube.com/watch?v=CgwY-1ddHic

https://www.youtube.com/watch?v=ACjyi4h--4Y

Once again thank you.

**Comment by rasteri** [comment]
Cool! I'll keep an eye out for that bug you mentioned. Let me know if you manage to catch it on video.

**Comment by schmitts** [comment]
With the new firmware, the scratch sample does not loop anymore, correct?

**Comment by 10shun** [comment]
> With the new firmware, the scratch sample does not loop anymore, correct?

Yeah it seems the scratch samples don't loop anymore. Is there a setting we can do make them loop again?

**Comment by rasteri** [comment]
No it doesn't loop anymore. I'll find a solution for that.
